### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by massazoum**
In this project I did these requirements,

- Create a new Rails app called 'hello-rails-react'.
- Use the command rails new -j webpack my_new_rails_app as described in the previous lesson's tutorial.
- Verify that the jsbundling-rails gem is added to your Gemfile.
- Initialize your project with Git.
- Install needed node dependencies as described in the previous lesson's tutorial.
- Create a babel.config.js in the root folder of your project as described in the previous lesson's tutorial.
- Update webpack.config.js to include the babel loader as described in the previous lesson's tutorial.
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
- Create a static view that will be the root of your app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Blog App <a name="about-project"></a>
+
+An excercice
```
Comment (FrederickMih) [inline README.md]: - In order to render our README file descriptive, let's kindly add a description of this project here.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+### Tech Stack <a name="tech-stack"></a>
+
+- Ruby
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **A login page**
+- [ ] **A registration page**
```
Comment (FrederickMih) [inline README.md]: - These are not the Key features of this project. Please take your time and properly document this README file. Let every information be in line with our project :+1:

File: README.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+<!-- ### Install -->
+
+Install this project with:
+```
+bundle install
+```
+
+
+### Usage
```
Comment (FrederickMih) [inline README.md:70]: - [ ] They should also be a command to set up the db:  rails db:create, rails db:migrate

File: README.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Deployment
+
+  The changes will be deployed to the website through our standard deployment process.
+- [Budget](https://budget-ixbx.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (FrederickMih) [inline README.md]: - [ ] This header has nothing to do with this current project. Let's kindly remove this header

**Review by FrederickMih — COMMENTED** [review]
## Hi :raising_hand: @massazoum, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not utilized the correct Gitflow as in the requirements. PR should be open from feature to dev branch.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by zainsadaqat** [comment]
# ❌ ❌ ❌Invalid Code Review Request ❌ ❌ ❌
# Hello @massazoum  👋  

## Something went wrong 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for this reason:

- ### You have submitted a project where you're using the WRONG Git Flow. As explained by the previous Code Reviewer, You have submitted a project, that does not utilize the correct Gitflow as in the requirements. PR should be open from the feature to the dev branch.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**